x-pack/legacy/plugins/reporting/server/lib/create_queue.js

```diff
@@ -7,31 +7,30 @@
 import { Esqueue } from './esqueue';
 import { createWorkerFactory } from './create_worker';
 import { oncePerServer } from './once_per_server';
-import { createTaggedLogger } from './create_tagged_logger';
-
-const dateSeparator = '.';
+import { LevelLogger } from './level_logger';
+import { createTaggedLogger } from './create_tagged_logger'; // TODO remove createTaggedLogger once esqueue is removed
 
 function createQueueFn(server) {
   const queueConfig = server.config().get('xpack.reporting.queue');
   const index = server.config().get('xpack.reporting.index');
-  const createWorker = createWorkerFactory(server);
 
-  const logger = createTaggedLogger(server, ['reporting', 'esqueue']);
   const queueOptions = {
     interval: queueConfig.indexInterval,
     timeout: queueConfig.timeout,
-    dateSeparator: dateSeparator,
+    dateSeparator: '.',
     client: server.plugins.elasticsearch.getCluster('admin'),
-    logger,
+    logger: createTaggedLogger(server, ['reporting', 'esqueue']),
   };
 
   const queue = new Esqueue(index, queueOptions);
 
   if (queueConfig.pollEnabled) {
     // create workers to poll the index for idle jobs waiting to be claimed and executed
+    const createWorker = createWorkerFactory(server);
     createWorker(queue);
   } else {
-    logger(
+    const logger = LevelLogger.createForServer(server, ['reporting', 'create_queue']);
```

Comment:
Should we capture these logging tags in a constant someplace? There's a lot of them getting peppered around now, and I worry that it'll be hard to reason about what tags to use without them being consolidated someplace.

Reply:
https://github.com/elastic/kibana/pull/41727/commits/fc3e83f75ea1dabc503fb3acc1a2f56591f6e018